commands/core/config.drush.inc

```diff
@@ -109,10 +109,8 @@ function config_drush_command() {
         'example-value' => 'branchname',
       ),
       'destination' => 'An arbitrary directory that should receive the exported files. An alternative to label argument.',
-      'skip-modules' => 'A list of modules to ignore during export (e.g. to avoid listing dev-only modules in exported configuration).',
```

Comment:
Maybe in the 8.x branch, we should leave the flag definition in place. This will give us a place to point at config-split, and also prevent breaking scripts with `--skip-modules=0`. If someone runs with `--skip-modules`, it should log a warning rather than failing.